test/traffic_ops_cfg/cfg_test.pl

```diff
@@ -239,20 +239,24 @@ sub get_crconfigs {
 	my $result     = &curl_me($to_cdn_url);
 	my $cdn_json   = decode_json($result);
 
+    my $old_ops = $CURL_OPTS;
+
 	foreach my $cdn ( @{ $cdn_json->{response} } ) {
 		next unless $cdn->{name} ne "ALL";
 		my $dir = $outpath . '/cdn-' . $cdn->{name};
 		make_path( $dir );
 		if ($perform_snapshot) {
 			print "Generating CRConfig for " . $cdn->{name};
 			my $start = [gettimeofday];
-			&curl_me( $to_url . "/tools/write_crconfig/" . $cdn->{name} );
+			$CURL_OPTS = $CURL_OPTS . ' -X PUT';
+			&curl_me( $to_url . "/api/2.0/snapshot/" . $cdn->{name} );
+            $CURL_OPTS = $old_ops;
```

Comment:
inconsistent indentation on this line

also, why the code changes? Why not just change the request path?

Reply:
The old endpoint is a GET, the new endpoint is PUT.